pkg/apis/operator/v1/types.go

```diff
@@ -145,6 +166,12 @@ type CalicoNetworkSpec struct {
 	// IPv6 addresses will not be auto-detected.
 	// +optional
 	NodeAddressAutodetectionV6 *NodeAddressAutodetection `json:"nodeAddressAutodetectionV6,omitempty"`
+
+	// HostPorts specifies if the plugin for HostPorts will be configured as part of the CNI configuration.
```

Comment:
I think this should be more use-facing - e.g. "HostPorts configures whether or not Calico will support Kubernetes HostPorts".

The plugin is an implementation detail, not API.